Project/MyGameLibrary/Enemy.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System.Drawing;
 
-namespace Fall2020_CSC403_Project.code {
-  /// <summary>
-  /// This is the class for an enemy
-  /// </summary>
-  public class Enemy : BattleCharacter {
+namespace Fall2020_CSC403_Project.code {
     /// <summary>
-    /// THis is the image for an enemy
-    /// </summary>
+    /// This is the class for an enemy
+    /// </summary>
+    public class Enemy : BattleCharacter {
+        /// <summary>
+        /// THis is the image for an enemy
+        /// </summary>
+    public static string[] enemyResponses = new string[5] { "Please, have mercy", "No. Stop.", "Ah", "Oh no", "How could you do this?" };

```

Comment:
There is a BattleCharacter class these objects inherit from, why not put a responses array in the BattleCharacter class "response"

Reply:
I chose to treat them as separate lists because one is full of attacker-specific dialogue the other contains defender-specific dialogue. I suppose you could combine them if you wanted to since currently the player and the enemy attack each other and take damage simultaneously.

Reply:
Right, however they are instance variables and can be defined in the child constructor see the attacks. Reference attacks implementation: Attacks is a list in the character class but enemies and players have unique attacks.